lib/fetch.ts

```diff
@@ -1,21 +1,42 @@
-import { dummyServiceMap } from "./dummies.ts";
 import { client, meta } from "./grpc.ts";
+import { GetAllResponse } from "snitch-protos/protos/external.ts";
+import { PipelineInfo } from "snitch-protos/protos/info.ts";
+import { dummyServiceMap } from "./dummies.ts";
+import {
+  FlowEdge,
+  FlowNode,
+  mapAudiencePipelines,
+  mapEdges,
+  mapNodes,
+} from "../components/serviceMap/customNodes.tsx";
+
+export type ServiceMapType = GetAllResponse & { pipes: PipelineInfo[] };
+
+export const getServiceMap = async (): Promise<ServiceMapType> => {
+  // const { response } = await client.getAll({}, meta);
```

Comment:
This service actually works...but I have dummy data shimmed in here for my layout testing. We can turn this off any time or come up with a better way to get test data...like a feature flag.